inst/specs/salaries.yml

```diff
@@ -0,0 +1,25 @@
+df:
+  source: Lahman::Salaries
+  name: salarios
+variables:
+  yearID:
+    trans: ano
+    desc: "Ano"
+  teamID:
+    trans: id_equipe
+    desc: ID da equipe (factor)
+  lgID:
+    trans: id_liga
+    desc: ID da liga (factor)
+  playerID:
+    trans: id_jogador
+    desc: ID do jogador
+  salary:
+    trans: salario
+    desc: "Salário em dólares"
+help:
+  name: salários
+  alias: salários
```

Comment:
@rivaquiroga  is there any problem to have accents in help: name and help:alias?

Reply:
Riva said it should be the same that in name. I'll change it! Thanks Ale!